.npmrc

```diff
@@ -0,0 +1 @@
+engine-strict=true
```

Comment:
I wonder if we could add a flag when running npm install as part of the CI instead of adding a dotfile? I'm just trying to avoid dotfiles if we can help it. New contributors will wonder what it is for, it adds clutter. The benefit might outweight the cost here, but there still is a cost, and I wonder if there is an alternative way to achieve the same thing?

Reply:
yeah, this one isnt as clear cut since the install command is hidden by the action. i'm open to ideas if you have thoughts.

i think the best alternative i can come up with would be to write to the file from the ci steps. would something along the lines of `echo 'engine-strict=true' >> .npmrc` before the install step be a reasonable middle ground?

Reply:
> i think the best alternative i can come up with would be to write to the file from the ci steps. would something along the lines of `echo 'engine-strict=true' >> .npmrc` before the install step be a reasonable middle ground?

let's do that, I like it

Reply:
updated to limit the check to the ci workflow